config/ms.yml

```diff
@@ -13,20 +14,33 @@ example_terms:
 - MS_0000000
 
 entries:
-- prefix: /releases/
-  replacement: https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v
-  tests:  
-    - from: /releases/4.1.86/psi-ms.owl
-      to: https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.86/psi-ms.owl
-    - from: /releases/4.1.86/psi-ms.obo
-      to: https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.86/psi-ms.obo
-
 - exact: /tracker
   replacement: https://github.com/HUPO-PSI/psi-ms-CV/issues
 
 - prefix: /about/
   replacement: https://psidev.info/groups/controlled-vocabularies
 
+- prefix: /home
+  replacement: https://github.com/HUPO-PSI/psi-ms-CV
+
+# intercept exact matches before version prefix blindly grabs everything
+- exact: /ms.owl
+  replacement: https://github.com/HUPO-PSI/psi-ms-CV/releases/latest/download/ms.owl
+- exact: /psi-ms.owl
```

Comment:
This is the right way to add "psi-ms.owl", which will resolve to <http://purl.obolibrary.org/obo/ms/psi-ms.owl>.